frontend/src/bundles/common/components/modal/modal.scss

```diff
@@ -0,0 +1,123 @@
+@import url("../../../../assets/css/variables.css");
+
+.modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100vw;
+    height: 100vh;
+    overflow-y: auto;
+    background-color: rgba(0 0 0 / 40%);
+    transform: scale(0);
+}
+
+.modal.active {
+    transform: scale(1);
+}
+
+.popup__close {
+    position: absolute;
+    top: 15px;
+    right: 20px;
+    font-weight: var(--font-weight-400);
+    font-size: var(--font-size-sub-heading);
+    background-color: transparent;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    opacity: 0.7;
+}
+
+.popup {
+    position: relative;
+    min-width: 500px;
+    min-height: 200px;
+    background-color: var(--color-white-100);
+    border-radius: 5px;
+}
+
+.modal-body {
+    padding: var(--p-1) var(--p-3);
+    color: var(--color-gray-300);
+    font-weight: var(--font-weight-400);
+    font-size: var(--font-size-level-2-body);
+    font-style: normal;
+}
+
+.modal-header {
+    display: flex;
+    width: 100%;
+    padding: var(--p-1) var(--p-3);
+    font-size: var(--font-size-level-6-heading);
+    font-style: normal;
+    line-height: var(--line-height-level-4-heading);
+}
+
+.modal-footer {
+    display: flex;
+    justify-content: end;
+    padding: var(--p-3) var(--p-3);
+}
+
+.modal-footer > * {
+    margin-left: var(--m-2);
+}
+
+.cancel {
+    width: 80px;
+    height: 40px;
+    color: var(--color-gray-300);
+    font-size: var(--font-size-1-button-text);
+    letter-spacing: var(--letter-spacing-button-text);
+    background: var(--color-white-100);
+    border: 1px solid var(--color-gray-300);
+    border-radius: 2px;
```

Comment:
Add to our css varibles

4px
8px
12px
16px

Reply:
do i also have to add 1px and 2px to variables or i can just keep it how it is?